docs/user-guides/workspace-access/jetbrains/jetbrains-toolbox.md

```diff
@@ -0,0 +1,65 @@
+# JetBrains Toolbox Integration
+
+JetBrains Toolbox helps you manage JetBrains products and includes remote development capabilities for connecting to Coder workspaces.
+
+## Before you begin
+
+- Install [JetBrains Toolbox](https://www.jetbrains.com/toolbox-app/) version 2.6.0.40284 or later
```

Comment:
the first 2.6 release was 2.6.0.40632

https://www.jetbrains.com/toolbox-app/download/other.html